src/modules/workSession/controller/workSession.controller.ts

```diff
@@ -55,7 +55,7 @@ export class WorkSessionController {
       dto,
     );
 
-    const statusCode = latestWorkSession.isUnfinished ? 200 : 204;
+    const statusCode = latestWorkSession.isUnfinished ? 204 : 200;
```

Comment:
This method always returns JSON so 204 doesn't apply I think. The status code here needs to indicate whether the new resource has been created or not.
So how about **201**?
https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/201

`const statusCode latestWorkSession.isUnfinished ? 200 : 201;`